.circleci/config.yml

```diff
@@ -273,6 +273,12 @@ workflows:
           target: manifests
           diff-fails: true
           requires: [prepare-env]
+      - go-make:
+          executor: ubuntu
```

Comment:
```suggestion
```
testing ci without a specified executor...

Reply:
oof, the `go` deps are present now, as expected, but because we run `make header-fix` after generating mocks, we also need our python deps. Not sure how to handle that, as they're on different executors. I see why you were planning to go with `ubuntu` here